docs/usage.md

```diff
@@ -0,0 +1,94 @@
+<OUTDIR>            # Finished results in specified location (defined with --outdir)
+.nextflow_log       # Log file from Nextflow
+# Other nextflow hidden files, eg. history of pipeline runs and old logs.
+```
+
+### Updating the pipeline
+
+When you run the above command, Nextflow automatically pulls the pipeline code from GitHub and stores it as a cached version. When running the pipeline after this, it will always use the cached version if available - even if the pipeline has been updated since. To make sure you are running the latest version of the pipeline, regularly update the cached version with:
+`nextflow pull Arcadia-Science/metagenomics`
+
+## Pipeline arguments
+
+Below lists several arguments to configure core parts of how Nextflow is run and pipeline-specific arguments you can modify.
+
+### Core Nextflow arguments
+
+These options are part of Nextflow and use a single hyphen.
+
+#### `-profile`
+
+Use this parameter to choose a configuration profile. Profiles can give configuration presets for different compute environments. Several generic profiles are bundled with the pipeline which instruct the pieline to use software packaged using different methods (Docker, Singularity, Conda, etc.) When using Biocontainers, most of the software packaging methods pull Docker containers from quay.io except for Singularity which directly downloads Singularity images via https hosted by the Galaxy project and Conda which downloads and installs software locally from Bioconda.
+
+We highly recommend the user of Docker or Singularity containers for full pipeline reproducibility, and have personally tested using Docker containers extensively within Arcadia. Other generic profiles besides the above listed are available, but they are untested and you can use at your own discretion.
+
+If `-profile` is not specified, the pipeline will run locally and expect all software to be installed and available on the `PATH` which is not recommended.
+
+- `docker`
+  - A generic configuration profile to be used with Docker
+- `singularity`
+  - A generic configuration profile to be used with Singularity
+- `conda`
+  - A generic configuration profile to be used with Conda. Please only use Conda as a last resort, such as when it is not possible to run with Docker, Singularity, or the other preset configurations.
+- `test`, `test_full`
+  - Profiles with a complete configuration for automated testing. This includes links to test data and need no other parameters
+
+#### `-resume`
+
+Specify this when restarting the pipeline from a previously failed run, updated part of the pipeline, or additional samples. Nextflow will use cached results from any pipeline steps where the inputs are the same, continuing from where it got to previously. For the input to be considered the same, not only the names of the files must be identical but all the files' contents as well.
+
+#### `-c`
+
+Specify the path to a specific config file.
```

Comment:
what does the config file have in it? do I have to specify this? If i don't, what file is the default?

Reply:
I believe this is if you have to provide an extra config for like an institution HPC or configs that aren't covered by default